3people/4week/p42889.py

```diff
@@ -0,0 +1,19 @@
+def solution(N, stages):
+  answer = []
+  fail = [[0,0] for _ in range(N)]
+  for stage in stages:
+    if stage == N+1:
```

Comment:
각 스테이지별로 도전한 사람을 갱신하는 과정에서 일일히 이전 스테이지를 갱신하는 것( 5번 스테이지 사람을 발견했을때 1 ~ 5단계 스테이지 도달자 값을 1씩 올려주는 것) 보다는
누적합의 개념을 활용해서 갱신할 수 있습니다. (일단 최종 도달 스테이지를 체크하고 뒷스테이지부터 누적합 적용)
ex)
```suggestion
    for stage in stages:
        arr[stage] += 1
    for i in range(len(arr)-1, 0,-1):
        arr[i-1] += arr[i]
```